### PR TITLE
Add link to jupyterhub-idle-culler

### PR DESCRIPTION
**Pull request by 1kastner**
See #584

 - [X] Add / update documentation

### CODE REVIEW

File: docs/topic/idle-culler.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ JupyterHub pings the user's notebook server at certain time intervals. If no res
 is received from the server during this checks and the timeout expires, the server is
 considered to be *inactive (idle)* and will be culled.
 
+The idle culler is maintained as
+`a separate project <https://github.com/jupyterhub/jupyterhub-idle-culler>`_.
+For advanced users, the code leaves the option for direct adjustment, e.g.  to add 
+features such as purging old user data.
```
Comment (GeorgianaElena) [inline docs/topic/idle-culler.rst:19]: What do you think about mentioning the `jupyterhub_config.py` escape-hatch here as a way of extending the culler config outside the `tljh-config's` scope? Maybe something like:

*The idle culler is enabled by default in TLJH and can be configured using `tljh-config`. For advanced use-cases, like purging old user data, the idle culler configuration can be extended beyond `tljh-config` options, using custom [`jupyterhub_config.py` snippets](https://tljh.jupyter.org/en/latest/topic/escape-hatch.html?highlight=escape-hatch#extending-jupyterhub-config-py)*

**Comment by 1kastner** [comment]
That sounds great to me. A slight change I'd prefer:

The idle culler is **installed and **enabled by default in TLJH and can be configured using tljh-config. For advanced use-cases, like purging old user data, the idle culler configuration can be extended beyond tljh-config options, using custom jupyterhub_config.py snippets

**Comment by GeorgianaElena** [comment]
It sounds great @1kastner :rocket:

**Comment by 1kastner** [comment]
@GeorgianaElena I hope I got it right in the RST document!

File: docs/topic/idle-culler.rst [hunk 2]
```diff
@@ -8,15 +8,15 @@ The idle culler automatically shuts down user notebook servers when they have
...
-
 JupyterHub pings the user's notebook server at certain time intervals. If no response
 is received from the server during this checks and the timeout expires, the server is
 considered to be *inactive (idle)* and will be culled.
 
-The idle culler is maintained as
-`a separate project <https://github.com/jupyterhub/jupyterhub-idle-culler>`_.
-For advanced users, the code leaves the option for direct adjustment, e.g.  to add 
-features such as purging old user data.
+The idle culler is a JupyterHub service that is installed and enabled by default in TLJH.
```
Comment (GeorgianaElena) [inline docs/topic/idle-culler.rst:15]:
```suggestion
The `idle culler <https://github.com/jupyterhub/jupyterhub-idle-culler>`_ is a JupyterHub service that is installed and enabled by default in TLJH.
```

**Comment by GeorgianaElena** [comment]
It looks good @1kastner (you can find the docs for this PR rendered here: https://the-littlest-jupyterhub--607.org.readthedocs.build/en/607/topic/idle-culler.html)
I just added a commit to re-add the link to the idle culler repo.

Merging this. Thanks! :heart:

**Comment by 1kastner** [comment]
Thank you very much!

**Pull request merged**